tests/cephadm/test_cluster_view.py

```diff
@@ -0,0 +1,108 @@
+from json import loads
+
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class ClusterViewError(Exception):
+    pass
+
+
+def verify_mon_service(host):
+    """
+    Verify MON service is up
+    Args:
+        host: Installer host
+    """
+    timeout, interval = 20, 6
+    for w in WaitUntil(timeout=timeout, interval=interval):
+        out = CephAdm(host).ceph.orch.ls(service_type="mon", format="json")
+        configs = loads(out)
+        for config in configs:
+            running = config.get("status").get("running")
```

Comment:
this raise an error in case status key is not present, it should be -
```
config.get("status", {}).get("running")
```

Reply:
Updated the changes.